### ISSUE TITLE
Not all content read on the menu screen with Talkback on

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#551]
Samsung Galaxy J3 Prime
Android version 7.0

Scenario to reproduce

While Talkback is on, using “Read from top” from “Global Context menu” or swiping right to move to each element on the COVID Alert menu screen, Talkback starts reading from the top of the screen. However, the screen doesn't scroll up automatically while Talkback is on. Talkback stops reading when it reaches what is visually displayed on the screen. 

Expected result
Talkback should read all the content on the screen.

Screenshots
![menu3](https://user-images.githubusercontent.com/12256067/86968072-88c30400-c139-11ea-8cd3-f537448e223c.gif)

**Comment by JuliannaR** [comment]
@smcmurtry @henrytao-me is this because of pageview vs scrollview? 
I can access the content under this point with android but it requires swiping to the right vs. read all.

**Comment by henrytao-me** [comment]
It's because of BottomSheet component that we are using. BottomSheet controls it own scroller which is not accessibility friendly. If we put a ScrollView inside it. It cannot scroll properly when there is screen reader.

**Comment by JuliannaR** [comment]
@henrytao-me lets test out your solution.

**Comment by JuliannaR** [comment]
Similar problem here: https://github.com/cds-snc/covid-shield-mobile/issues/550

**Comment by henrytao-me** [comment]
Ref https://github.com/cds-snc/covid-shield-mobile/pull/576

**Comment by qanitah** [comment]
Android 8.1.0 
This is still happening in version 1.0 (49) and user is stuck at the bottom of the visual screen. TalkBack stops announcing here for swiping right and "read from top".

Android 10 on OnePlus 7 is fine.

![image](https://user-images.githubusercontent.com/14280062/87557095-7db22b80-c685-11ea-8372-fcad4f48dcad.png)

**Comment by sarasuri** [comment]
Samsung Galaxy J3 Prime
Android version 7.0
Application version: 1.0 (49)

The issue still remains the version 49.
